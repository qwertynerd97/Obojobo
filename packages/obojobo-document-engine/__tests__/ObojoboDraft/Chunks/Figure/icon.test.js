import React from 'react'
import renderer from 'react-test-renderer'

import Icon from 'ObojoboDraft/Chunks/Figure/icon'

describe('Figure Icon', () => {
	test('Icon', () => {
		const component = renderer.create(<Icon />)
		const tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})
})
