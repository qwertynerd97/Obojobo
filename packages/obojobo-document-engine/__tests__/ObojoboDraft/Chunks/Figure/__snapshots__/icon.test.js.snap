// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Figure Icon Icon 1`] = `
<svg
  data-name="Layer 1"
  id="Layer_1"
  viewBox="0 0 21.61 16.3"
  xmlns="http://www.w3.org/2000/svg"
>
  <defs>
    <style>
      .cls-1{opacity:0.55;}
    </style>
  </defs>
  <title>
    figure
  </title>
  <g
    className="cls-1"
  >
    <path
      d="M1.2,3.85v16.3H22.8V3.85H1.2ZM21.91,19.26H18.55A1.65,1.65,0,0,0,18.38,19l-4.72-4.72a1.68,1.68,0,0,0-2.37,0L10.61,15,6.68,11.08a1.67,1.67,0,0,0-2.37,0L2.09,13.31V4.74H21.91V19.26Z"
      transform="translate(-1.2 -3.85)"
    />
    <circle
      cx="15.55"
      cy="5.98"
      r="2.49"
    />
  </g>
</svg>
`;
